lib/src/browser/disposable_browser.dart

```diff
@@ -198,14 +199,20 @@ class Disposable implements disposable_common.Disposable {
   void manageDisposable(disposable_common.Disposable disposable) =>
       _disposable.manageDisposable(disposable);
 
+  @deprecated
   @override
   void manageDisposer(disposable_common.Disposer disposer) =>
```

Comment:
Add a doc comment identifying the migration path now that this method is deprecated.